src/components/shared/Dropdown.css

```diff
@@ -10,17 +10,19 @@
   z-index: 1000;
 }
 
+.dropdown-block {
+  align-self: flex-end;
```

Comment:
is this element in a flex container ? If so, I think we could remove the `position: relative`